### PR TITLE
new pattern

### PR DESCRIPTION
**Pull request by sabornibhattacharya**
*Issue #, if available:*

*Description of changes:*

This sample project demonstrates how to use an API Gateway REST API as a proxy to Comprehend DetectSentiment API. You can easily utilise this solution to expose other Comprehend API operations as well with minimal efforts.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: apigw-comprehend/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        "LanguageCode": "en",
+        "Text": "Bob lives in Seattle. He is a software engineer at Amazon."
+    }'
+```
+
+You can also run the following commands on a terminal prompt to get the API Endpoint, Job Queue and Job definition.
+
+**API Endpoint**
+```
+aws cloudformation describe-stacks --stack-name [YOUR STACK NAME] --query "Stacks[0].Outputs[?OutputKey=='ApiEndpoint'].OutputValue" --output text
```
Comment (pputhran) [inline apigw-comprehend/README.md:56]: "Stacks[0].Outputs[?OutputKey=='ComprehendApiEndpoint']

File: apigw-comprehend/README.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+```
+curl --location --request POST '[YOUR API ENDPOINT]' --header 'Content-Type: application/json' \
+--data-raw '{
+        "LanguageCode": "en",
+        "Text": "Bob lives in Seattle. He is a software engineer at Amazon."
+    }'
+```
+
+You can also run the following commands on a terminal prompt to get the API Endpoint, Job Queue and Job definition.
```
Comment (pputhran) [inline apigw-comprehend/README.md:52]: Job Queue and Job Definition can be removed.

File: apigw-comprehend/example-pattern.json [hunk 3]
```diff
@@ -0,0 +1,57 @@
+{
+  "title": "API Gateway to Comprehend",
+  "description": "Create a REST API that acts as a proxy to Comprehend DetectSentiment API operation.",
+  "language": "Python",
```
Comment (pputhran) [inline apigw-comprehend/example-pattern.json:4]: Should be updated to ""

File: apigw-comprehend/example-pattern.json [hunk 4]
```diff
@@ -0,0 +1,57 @@
+{
+  "title": "API Gateway to Comprehend",
```
Comment (pputhran) [inline apigw-comprehend/example-pattern.json:2]: Amazon API Gateway to Amazon Comprehend

File: apigw-comprehend/example-pattern.json [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+  "description": "Create a REST API that acts as a proxy to Comprehend DetectSentiment API operation.",
+  "language": "Python",
+  "level": "200",
+  "framework": "SAM",
+  "introBox": {
+    "headline": "How it works",
+    "text": [
+      "This sample project demonstrates how to use an API Gateway REST API as a proxy to Comprehend DetectSentiment API.",
```
Comment (pputhran) [inline apigw-comprehend/example-pattern.json:10]: Amazon API Gateway and Amazon Comprehend

**Review by pputhran — CHANGES_REQUESTED** [review]
A few suggestions

File: apigw-comprehend/README.md (hunk 1 above)
Comment (sabornibhattacharya) [reply apigw-comprehend/README.md]: It seems the file didn't get saved while editing in VS Code. Changes updated

File: apigw-comprehend/README.md (hunk 2 above)
Comment (sabornibhattacharya) [reply apigw-comprehend/README.md]: It seems the file didn't get saved while editing in VS Code. Changes updated

File: apigw-comprehend/example-pattern.json (hunk 3 above)
Comment (sabornibhattacharya) [reply apigw-comprehend/example-pattern.json]: Done

File: apigw-comprehend/example-pattern.json (hunk 4 above)
Comment (sabornibhattacharya) [reply apigw-comprehend/example-pattern.json]: Done

File: apigw-comprehend/example-pattern.json (hunk 5 above)
Comment (sabornibhattacharya) [reply apigw-comprehend/example-pattern.json]: Done

**Review by pputhran — APPROVED** [review]

**Pull request merged**